pages/common/swagger-codegen.md

```diff
@@ -0,0 +1,19 @@
+# swagger-codegen
+
+> Generate code and documentation for your REST api from a OpenAPI/swagger definition
+
+- Generate from a file or an url
+
+`swagger-codegen generate --input-spec {{url_or_file}} ---lang {{language}} --library {{library}}`
+
+- Available languages:
+
+ `swagger-codegen langs`
+
+- Available configuration options
+
+ `swagger-codegen config-help -l java`
+
+- Help
```

Comment:
How about `Display help options for the generate command`. And also put generate in tokens.